toolchain/check/inst_block_stack.cpp

```diff
@@ -21,6 +21,11 @@ auto InstBlockStack::Push(SemIR::InstBlockId id) -> void {
   ++size_;
 }
 
+auto InstBlockStack::PushInit() -> void {
+  Push(SemIR::InstBlockId::Inits);
+  stack_[size_ - 1].content = init_block_;
```

Comment:
```suggestion
  stack_[size_ - 1].content = std::move(init_block_);
```

At present, this would perform a copy on PushInit and PopInit. That would create scaling issues when there are many or complex initializers. Using `move` should alleviate the issue.

Reply:
Done!